drivers/sensor/stts751/stts751.c

```diff
@@ -201,15 +200,14 @@ static int stts751_init(const struct device *dev)
 static struct stts751_data stts751_data;
 
 static const struct stts751_config stts751_config = {
-	.master_dev_name = DT_INST_BUS_LABEL(0),
+	.i2c = I2C_DT_SPEC_INST_GET(0),
```

Comment:
this is set unconditionally, `i2c` field will not always be available

Reply:
Oh, thanks!